src/timing.jl

```diff
@@ -22,31 +21,20 @@ end
 #     freq: the maximum frequency at which fpswhen should update
 # Returns:
 #     an signal of Float64 time deltas
+function gate(wason_timer::Tuple{Bool, Timer}, ison::Bool, s::Input{Float64}, delta::Float64)
+    wason, timer = wason_timer
+    (!wason&&ison) && return (ison, Timer(x->push!(s, time()), 0, delta)) # start pushing again
```

Comment:
true - I started doing this to make it clear which part is the condition and which is the statement to execute.
Some of my weakly enforced coding standards you could say...